packages/components/src/input-control/styles/input-control-styles.tsx

```diff
@@ -260,72 +317,3 @@ export const Label = (
 export const LabelWrapper = styled( FlexItem )`
 	max-width: calc( 100% - 10px );
 `;
-
-type BackdropProps = {
-	disabled?: boolean;
-	isBorderless?: boolean;
-	isFocused?: boolean;
-};
-
-const backdropFocusedStyles = ( {
-	disabled,
-	isBorderless,
-	isFocused,
-}: BackdropProps ): SerializedStyles => {
-	let borderColor = isBorderless ? 'transparent' : COLORS.ui.border;
-
-	let boxShadow;
-	let outline;
-	let outlineOffset;
-
-	if ( isFocused ) {
-		borderColor = COLORS.ui.borderFocus;
-		boxShadow = CONFIG.controlBoxShadowFocus;
-		// Windows High Contrast mode will show this outline, but not the box-shadow.
-		outline = `2px solid transparent`;
-		outlineOffset = `-2px`;
-	}
-
-	if ( disabled ) {
-		borderColor = isBorderless ? 'transparent' : COLORS.ui.borderDisabled;
```

Comment:
The important priority logic in here is for `border-color`, where `isFocused` > `isBorderless` > `disabled` (accounting for the fact that a disabled control will never be focused). This priority logic still holds true in the refactored styles.